### PR TITLE
Card

### PR DESCRIPTION
**Pull request by edouardmisset**
I created a simple card so that we can see the student's profile...

### CODE REVIEW

File: src/components/Card.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  1. Global styles
+  2. typography
+  3. Layout
+  4. Components
+*/
+
+/* ================
+  Global variables
+  ================*/
+:root {
```
Comment (ComicScrip) [inline src/components/Card.css:12]: A a voir si on bouge ça dans un fichier global, histoire d'harmoniser les choses sur toute l'application.
Il faudrait qu'on discute ensemble d'une stratégie pour le style de l'app. 
Peut-être partir sur https://material-ui.com/ ou/et https://tailwindcss.com/  ?

**Comment by edouardmisset** [comment]
Absolutely.

**Comment by ComicScrip** [comment]
I like the idea of a Card. 

image

I think we should make one popup when we hover/click on an Avatar anywhere in the app in order to give detailed information about the person and also useful links (Meet, Discord, GitHub, ...).

There's a library called [Popper](https://popper.js.org/) that helps dealing with popovers. I know there's a react version of the latter (but never tried it). Maybe we could give it a chance !
Otherwise there's MUI's Popover : https://material-ui.com/components/popover/ 

Styling is still to be discussed in order to make the Card component fit well into the app.
What do you think ?

**Comment by edouardmisset** [comment]
I agree :)

**Pull request closed without merging**